spec/03-plugins/36-request-transformer/02-access_spec.lua

```diff
@@ -859,154 +879,6 @@ describe("Plugin: request-transformer(access) [#" .. strategy .. "]", function()
     end)
   end)
 
-  describe("rename", function()
```

Comment:
line 862-1009 seems to be duplicated with line 699-844, so remove them 

https://github.com/Kong/kong/blob/c33d745793b454ab27d9681562fadbccbc454a1b/spec/03-plugins/36-request-transformer/02-access_spec.lua#L862-L1009

https://github.com/Kong/kong/blob/c33d745793b454ab27d9681562fadbccbc454a1b/spec/03-plugins/36-request-transformer/02-access_spec.lua#L699-L844